packages/gatsby-theme-learning/src/hooks/use-course-details.ts

```diff
@@ -39,7 +39,14 @@ export const useFetchCourseDetails = (
 
   const { data, error, isLoading } = useSWR(
     shouldFetchData ? apiEndpoint : null,
-    (url) => fetcherWithToken(url, getAuthToken, learnApiBaseUrl, 'GET')
+    (url) => fetcherWithToken(url, getAuthToken, learnApiBaseUrl, 'GET'),
+    {
+      revalidateOnReconnect: false,
+      revalidateIfStale: false,
+      revalidateOnMount: true,
+      dedupingInterval: 10000000,
+      revalidateOnFocus: false,
```

Comment:
@gabriele-ct  I would prefer to leave this set to "true" - it's a really good way to prevent user confusion when they work in multiple browser tabs when taking a course.  I had understood you only turned it on to have less noise when debugging.  

Similar but less clearly a use case for "revalidateOnReconnect" - that is rather a sign of the computer having gone to sleep in the meanwhile which is a good proxy for a user returning after a longer pause.